src/main/java/org/sonarlint/intellij/util/HelpLabelUtils.kt

```diff
@@ -22,16 +22,30 @@ package org.sonarlint.intellij.util
 import com.intellij.ide.BrowserUtil
 import com.intellij.ui.ContextHelpLabel
 import org.sonarlint.intellij.documentation.SonarLintDocumentation.Intellij.FOCUS_ON_NEW_CODE_LINK
+import org.sonarlint.intellij.documentation.SonarLintDocumentation.Marketing.SONARCLOUD_PRODUCT_SIGNUP_LINK
 import org.sonarlint.intellij.telemetry.LinkTelemetry
 
 class HelpLabelUtils {
     companion object {
 
         @JvmStatic
-        fun createCleanAsYouCode() =
+        fun currentFileHelp() =
                 ContextHelpLabel.createWithLink(null,
-                        "Deliver clean code by focusing on code that was recently modified",
-                        "Learn more about Clean as You Code", true) { BrowserUtil.browse(FOCUS_ON_NEW_CODE_LINK) }
+                        "SonarLint analyzes in real-time the active file only and reports any issues found in this view. " +
+                            "SonarCloud complements SonarLint by detecting more across the whole project.",
+                        "Try SonarCloud for free", true) { BrowserUtil.browse(SONARCLOUD_PRODUCT_SIGNUP_LINK) }
```

Comment:
Shouldn't you have the link on `currentFileHelpNotConnected` instead of `currentFileHelp`?